project-koku/koku
# Generate org_admin authorization header for source status pushes.

Addresses #2125 

See issue for background on this as well as recent discussion on the sources slack channel.  For now I'm going to fix just the availability checker status push path.  The platform is open to changes around how services communicate internally to address concerns around relying on RBAC status of the user that created the source for handling source-application events.

**Testing**
1. Create AWS source with bad bucket
2. Request status from our internal endpoint with a similar header to what the platform is using.
3. Verify status was received successfully in platform sources.

**Test Results**

[source_status_header_ut.txt](https://github.com/project-koku/koku/files/4593706/source_status_header_ut.txt)
